recipes/omniscidb-cuda/recipe/meta.yaml

```diff
@@ -0,0 +1,103 @@
+{% set name = "omniscidb-cuda" %}
+{% set version = "4.8.1" %}
+{% set sha256 = "300cdbbaa9108879f7ec22d9cf6210ca2700a3ef99c356a5c00fbf3aeb3a4713" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/omnisci/omniscidb/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    Skip-installing-Boost-libs.patch
+
+build:
+  number: 0
+  skip: True  # [not (linux64 and cuda_compiler_version == "10.1")]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler("cgo") }}
+    - {{ compiler("cuda") }}
+    #- {{ compiler('c') }}
+    # omniscidb 4.8 is not boost-cpp > 1.68 ready
+    - boost-cpp 1.68.*        # to avoid cmake picking up boost-cpp 1.70+
+    - cmake
+    - make
+    - maven
+    # cmake uses lscpu, bc
+    - util-linux  # required for lscpu
+    - bc
+  host:
+    - {{ compiler('cxx') }}
+    - {{ compiler("cuda") }}
+    #- {{ compiler('c') }}
+    - boost-cpp 1.68.*
+    # omniscidb 4.8 is not llvm > 9 ready
+    - llvmdev <9
+    - clangdev <9
+    - double-conversion
+    - snappy
+    - gflags
+    - glog
+    - libarchive
+    - libkml
+    - libpng
+    - blosc
+    - libgdal >=2.3
+    - arrow-cpp >=0.12.1
+    - thrift-cpp >=0.11.0
+    - ncurses
+    - flex
+    - bisonpp
+    - openssl
+    - librdkafka
+    - openldap
+    # Required by cudatoolkit-dev:
+    - wget
+  run:
+    - boost-cpp 1.68.*
+    - arrow-cpp >=0.12.1
+    - snappy
+    - double-conversion
+    - libgdal >=2.3
+    - glog
+    - gflags
+    - openjdk 8.*
+    - ncurses
+    - xz
+    - bzip2
+    - zlib
+    # omnscidb UDF support requires C++ headers
+    - {{ compiler('cxx') }}
+    - llvmdev <9
+    - clangdev <9
```

Comment:
Do you need the llvm libraries or just `clang` at runtime?

Reply:
Checked locally, llvmdev can be removed.

Reply:
Fixed.